README.md

```diff
@@ -190,6 +190,18 @@ Zino will check `polldevs.cf` for changes on a scheduled interval while it's
 running, so any changes made while Zino is running should be picked up without
 requiring a restart of the process.
 
+### Configuring other settings
+
+Other settings can be also configured in a separate [TOML](https://toml.io/en/) file,
+which defaults to `zino.toml` in the current working directory, but a different file
+can be specified using the `--config-file` command line option.
```

Comment:
This is according to spec, but it's not what is currently happening in this PR:

If I create `zino.toml`, I expect Zino to read it without me specifying the `--config-file` option.  If it is missing, I expect Zino to ignore it and run with the defaults.

Currently, if I add a `./zino.toml` and run Zino without `--config-file`, Zino ignores my config.